### ISSUE TITLE
base16 ocean dark request

### ISSUE THREAD

**Issue by gavsiu** [Qeole/Enlight#4]
I'm using Base16 Ocean Dark colors. Can you please add this seeing as you have Base16 Tomorrow. Thanks!

**Comment by Qeole** [comment]
Hi @gavsiu, and thanks for using the add-on!

I would be glad to add this color scheme, but it it not possible to do it here: I do not “choose” or “define” the schemes for the add-on, it just embeds all the styles provided by the JavaScript library it relies on (namely [highlight.js](https://highlightjs.org/)). So your style will automatically be available here once it gets merged into a new version of this lib, and the lib into Enlight.

Thus I encourage you to submit the issue on [the highlight.js GitHub repository](https://github.com/isagalaev/highlight.js/issues/new) or, if you feel like it, [to create it](https://highlightjs.readthedocs.io/en/latest/style-guide.html) (the existing styles being [there](https://github.com/isagalaev/highlight.js/tree/master/src/styles)) and to submit a pull request to highlight.js project.

**Comment by gavsiu** [comment]
I thought it would be faster if I do it myself. First time forking and submitting a pull request.

https://github.com/isagalaev/highlight.js/pull/1188

I'll let you know if it's accepted.

**Comment by Qeole** [comment]
@gavsiu congrats! Welcome to the world of open-source :-)

Once your style is accepted we need to wait for the next stable version of highlight.js to be released (usually every two months or so, the last one being on May 17th).

**Comment by Qeole** [comment]
@gavsiu I've seen that your pull request have just been merged. Congrats again! Now we are waiting for the next stable release.

**Comment by Qeole** [comment]
[highlight.js v9.5.0](https://github.com/isagalaev/highlight.js/releases/tag/9.5.0), [which includes the new style sheet](https://highlightjs.org/#news-95), has just been released. I will try to update the add-on this weekend.

**Comment by Qeole** [comment]
Fixed with commit 0cad74a0d4b220be9ba35eec3a49d57b720fb1b5. I did not pushed it upstream immediately because I faced [this issue](https://github.com/isagalaev/highlight.js/issues/1235) on highlight.js, which broke the add-on. Now it's fixed, and highlight.js v9.6.0 even came out today, so I repacked and pushed to Mozilla's platform. It should be live in a few days.

**Comment by gavsiu** [comment]
Not to worry. I stopped using Arch and the Ocean theme. Lately I've been using the RecognEyes theme in my IDE and ported it to Firefox's view-source. I've just been too lazy to port it to highlight.js.

**Comment by Qeole** [comment]
Ok great! By the way I did not know you could select themes for the view-source function in Firefox. If you don't mind me asking, how would you do that?

**Comment by gavsiu** [comment]
You change it by adding it to [userContent.css](http://kb.mozillazine.org/UserContent.css#Creating)

View-source css file can be viewed here for your version of Firefox, but I doubt it has changed much: resource://gre-resources/viewsource.css

I copied everything and deleted the `@charset` and `@namespace`. Then I deleted everything else I didn't need to keep the file smaller.

Then you wrap all that css with

    @-moz-document url-prefix(view-source:) {
        ...copied css...
    }

so that it won't try to style every website you visit (checks for `view-source:` in the url).

Also, not present in the source is highlight color, so I added my own.
